pkg/log/log.go

```diff
@@ -0,0 +1,54 @@
+package ncolog
```

Comment:
usually pkg name matches the folder.

why not call this one log ?

Reply:
also do you need this pkg ?

main reason to have it is to support runtime updates of log level.

that said its fine to have it for intialization of zap logger.

Reply:
Changed the folder name to match the package